.ado/scripts/prepublish-check.mjs

```diff
@@ -48,7 +48,7 @@ function isMainBranch(branch) {
  * @param {string} branch
  */
 function isStableBranch(branch) {
-  return /^\d+\.\d+-stable$/.test(branch);
+  return /\d+\.\d+-stable$/.test(branch);
```

Comment:
`refs/heads/` was supposed to be stripped out in `getCurrentBranch()` further down. Can you check whether that part is wrong?

Reply:
Ah, I see it. Unfortunately, PR branches just have the name "merge", so to check I think I'll have to run the pipeline off a branch created on the repo. I'll do that tomorrow morning.

Reply:
This was fine, I scrapped the whole regex thing and repurposed this PR.